From cc5c2a587045949f292f185116273a9be2069a9b Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke@hauke-m.de>
Date: Sun, 4 Oct 2026 01:55:38 +0200
Subject: [PATCH 138/138] rpcd: update to Git HEAD (2026-10-04)

d99f703e4035 session: stop naming runtime files after the session id

Link: https://github.com/openwrt/openwrt/pull/25587
(cherry picked from commit 09538fccfcc1ec4d6001512c7136d7aae12a8bc2)
Link: https://github.com/openwrt/openwrt/pull/25608
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
---
 package/system/rpcd/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/package/system/rpcd/Makefile b/package/system/rpcd/Makefile
index e1ab4be6c0..0c6f6d0589 100644
--- a/package/system/rpcd/Makefile
+++ b/package/system/rpcd/Makefile
@@ -12,9 +12,9 @@ PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL=$(PROJECT_GIT)/project/rpcd.git
-PKG_MIRROR_HASH:=c07273d63b99a8f05fb4a99d1113908a066dd2ff3151662b1fb7ef2ab7c1828c
-PKG_SOURCE_DATE:=2026-09-25
-PKG_SOURCE_VERSION:=a6c6b63b47bd8f4f762ef2e6787210cd3e0fb8a0
+PKG_MIRROR_HASH:=01eed566d6d5f54460225ccfa93a572bca66ca8b1458b947060bf00d360f65b4
+PKG_SOURCE_DATE:=2026-10-04
+PKG_SOURCE_VERSION:=d99f703e4035d03fbcfbada3cb697db346c2b984
 PKG_MAINTAINER:=Jo-Philipp Wich <jo@mein.io>
 
 PKG_LICENSE:=ISC
-- 
2.34.1

